Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -225,7 +225,12 @@ def print_caveats
   def enable_accessibility_access
     return unless @cask.accessibility_access
     ohai "Enabling accessibility access"
-    if MacOS.version <= :mountain_lion
+    if MacOS.version >= :sierra
+      opoo <<-EOS.undent
+        Access denied to accessibility DB (TCC.db) by SIP on this version of macOS.
+        If needed, enable manually via the app or System Preferences.
+      EOS
+    elsif MacOS.version <= :mountain_lion
```

Comment:
How about
```
Accessibility access cannot be enabled automatically on this version of macOS.
See System Preferences to enable it manually.
```
? Is there a line length it should not exceed (i.e. should I wrap 'macOS' to the next line)?